modules/hooks.nix

```diff
@@ -18,6 +18,14 @@ in {
 
   config.pre-commit.hooks =
     {
+      ansible-lint =
+        {
+          name = "ansible-lint";
+          description =
+            "Ansible linter";
+          entry = "${tools.ansible-lint}/bin/ansible-lint";
+          files = "\\.yml$";
```

Comment:
Technically it should be "\\.(yaml|yml)$" but I am not sure what is possible here ?

Reply:
I think this should be left to the user, to point it at the right files that are actually ansible files.
The [ansible's .pre-commit-hooks.yaml](https://github.com/ansible/ansible-lint/blob/master/.pre-commit-hooks.yaml) also leaves it out.